spyder/plugins/editor/widgets/tests/fixtures.py

```diff
@@ -19,11 +19,31 @@
 from qtpy.QtGui import QFont
 
 # Local imports
+from spyder import version_info
 from spyder.plugins.editor.widgets.codeeditor import CodeEditor
 from spyder.plugins.editor.widgets.editor import EditorStack
 from spyder.widgets.findreplace import FindReplace
 
 
+@pytest.fixture
+def code_editor_bot(qtbot):
```

Comment:
Tests are failing because there is another `code_editor_bot` fixture in `conftest.py` but with different content. Let me do a quick PR to try to improve the situation and then you can rebase in top of master.